FileUpload/src/FileUpload.jsx

```diff
@@ -251,9 +251,9 @@ export default class FileUpload extends Component {
         this.clearFileUploaderValue(this.refs.fileInput2);
     }
 
-    clearFileUploaderValue(fileInput){
-        if(fileInput){
-            fileInput.value = '';
+    clearFileUploaderValue(fileInput) {
+        if (fileInput) {
+            fileInput.value = "";
```

Comment:
I though single quotes were the default ;)

Reply:
project standard seems to be double quotes :)